sidechain/storage/src/lib.rs

```diff
@@ -22,6 +29,12 @@ mod error;
 pub mod interface;
 mod storage;
 
+#[cfg(test)]
+mod storage_tests_get_blocks_following;
```

Comment:
I created a separate file for the new tests, since the original `storage.rs` file is IMO already way beyond anything reasonable in size.